app/javascript/shared/components/tags.jsx

```diff
@@ -362,6 +364,7 @@ class Tags extends Component {
     const { searchResults, selectedIndex, showingRulesForTag } = this.state;
     const { classPrefix, defaultValue, maxTags, listing } = this.props;
     const searchResultsRows = searchResults.map((tag, index) => (
+      // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
```

Comment:
this shouldn't be disabled though...

Reply:
`<div
        tabIndex="-1"
        role="button"
        className={'${classPrefix}__tagoptionrow ${classPrefix}__tagoptionrow--${
          selectedIndex === index ? 'active' : 'inactive'
        }'}
        onClick={this.handleTagClick}
        onKeyDown={this.handleTagClick}
        data-content={tag.name} >
`
if i fix with the code will leads to an UI error.
when we select tag using keydown, it won't works :-)
so that i marked as lint fix

Reply:
Are you sure?

If you google the names of the errors you'll find hints on how to solve them:

- [jsx-a11y/click-events-have-key-events](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md)
- [jsx-a11y/no-static-element-interactions](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md)

Reply:
changes updated @rhymes